src/Other/Helpers/Reflection.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rubix\ML\Other\Helpers;
+
+use ReflectionClass;
+use ReflectionException;
+
+/**
+ * Reflection
+ *
+ * Utility functions relying on PHP's runtime reflection API
+ *
+ * @see https://www.php.net/manual/en/book.reflection.php
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Chris Simpson
+ */
+class Reflection
+{
+    /**
+     * Property.
+     *
+     * Access a class property even if it's visibility is not public.
+     * This is typically useful for debugging and creating informative logs.
+     *
+     * @param object $subject The object to introspect
+     * @param string $name The property to access
```

Comment:
Same sentiment as above but in addition this comment is sorta misleading. It is not the property but the *name* of the property to access.